periodic/periodic.go

```diff
@@ -488,6 +489,41 @@ func (r *periodicRunner) Run() RunnerResults {
 	return result
 }
 
+type fileAccessLogger struct {
+	mu     sync.Mutex
+	file   *os.File
+	format string
+}
+
+// AccessLogger defines an interface to report a single request.
+type AccessLogger interface {
+	// Report logs a single request to a file.
+	Report(thread int, time int64, latency float64)
+}
+
+// NewFileAccessLogger creates an AccessLogger that writes to the provided file in the provided format.
+func NewFileAccessLogger(file, format string) (AccessLogger, error) {
+	f, err := os.OpenFile(file, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0o644)
+	if err != nil {
+		return nil, err
+	}
+	return &fileAccessLogger{file: f, format: format}, nil
```

Comment:
format isn't validated... so it'll fail silently/weirdly later?

I would maybe log both errors here and then in caller (main) you can just
```
  // error already logged
  os.Exit(1)
```
instead of the log.Fatalf... I mentioned there

Reply:
done